docs/working-with-argocd.md

```diff
@@ -73,24 +71,21 @@ You can sync your applications via UI by simply clicking the SYNC button and fol
 
 ![](./images/argocd-sync-apps.png)
 
-More details please refer to [argocd guide: sync the application](https://argo-cd.readthedocs.io/en/stable/getting_started/#7-sync-deploy-the-application).
+For more details, please refer to the [Argo CD guide: sync the application](https://argo-cd.readthedocs.io/en/stable/getting_started/#7-sync-deploy-the-application).
 
 ## Checking Apps Status
-For deployment running in more than one clusters, you don't need to create applications for each 
-cluster. You can get the overall and detailed status from one `Application`.
+For deployment running in more than one clusters, you don't need to create applications for each cluster. You can get the overall and detailed status from one `Application`.
 
 ![](./images/argocd-status-overview.png)
 
-The `svc/guestbook-ui`, `deploy/guestbook-ui` and `propagationpolicy/guestbook` in the middle of the picture are the 
-resources created by the manifest in the forked repo. And the `resourcebinding/guestbook-ui-service` and 
-`resourcebinding/guestbook-ui-deployment` in the right of the picture are the resources created by Karmada.
+The `svc/guestbook-ui`, `deploy/guestbook-ui` and `propagationpolicy/guestbook` in the middle of the picture are the resources created by the manifest in the forked repo. And the `resourcebinding/guestbook-ui-service` and `resourcebinding/guestbook-ui-deployment` on the right of the picture are the resources created by Karmada.
 
 ### Checking Detailed Status
 You can obtain the Deployment's detailed status by `resourcebinding/guestbook-ui-deployment`. 
 
 ![](./images/argocd-status-resourcebinding.png)
 
 ### Checking Aggregated Status
-You can obtain the aggregated status of the Deployment from UI by `deploy/guestbook-ui`.
+You can obtain the aggregated status of the deployment from UI using `deploy/guestbook-ui`.
```

Comment:
Not sure which style you prefer for describing a stateless app.
Sometimes you capitalize **d**, sometime you don't. 
From my point of view, I will stick to **D**eployment.